bigbluebutton-html5/private/locales/en.json

```diff
@@ -178,6 +178,7 @@
     "app.presentationUploder.conversion.generatedSlides": "Slides generated ...",
     "app.presentationUploder.conversion.generatingSvg": "Generating SVG images ...",
     "app.presentationUploder.conversion.pageCountExceeded": "Ops, the page count exceeded the limit of 200 pages",
+    "app.presentationUploder.conversion.pdfHasBigPage": "We could not convert the PDF file, please try optimizing it",
```

Comment:
I think we're going to need to change the string. I doubt most users would know what to do to rectify the problem if they were shown this message.

Reply:
Sure, it is a technical advice. It needs a wording improvement.